draft-ietf-core-sid-latest.md

```diff
@@ -517,6 +517,54 @@ The allocation policy is Expert review. The Expert MUST ensure that the followin
   the same SIDs as in the the other ".sid" file.
 * SIDs never change.
 
+* IANA is expected to store the SID files and make them available along with the associated YANG files in the YANG Module Names registry.   
+
+* SID files SHOULD be included in documents using the same CODE BEGINS mechanism that is used for attaching YANG modules
+
+### Recursive Allocation of SID Range at Document Adoption time
+
+Due to the difficulty in changing SID values during IETF document processing, it is expected that most documents will ask for SID allocations using Early Allocations {{-BCP100}}.   
+The details of the Early Allocation should be included in any Working Group Adoption call.
+
+During the early use of SIDs, many existing, previously published YANG modules will not have SID allocations.  
+For an allocation to be useful the included YANG modules may also need to have SID allocations made.
+
+The Expert Reviewer who performs the (Early) Allocation analysis will need to go through the list of included YANG modules and perform SID allocations for those modules as well.
+
+If the document is a published RFC, then the allocation is permanent.
```

Comment:
I am not sure it is clear enough what is permanent - the SID allocation I suppose, so this is what is added in my version. Also formatted as bullet points in order to connect tightly coupled paragraphs together IMO improving readability